server/src/repositories/Equipment/EquipmentRepository.ts

```diff
@@ -83,38 +85,38 @@ export async function updateTrainingModules(
 }
 
 export async function updateEquipment(
-  id: number,
-  equipment: EquipmentInput
+    id: number,
+    equipment: EquipmentInput
 ): Promise<Equipment | null> {
   await knex("Equipment")
-    .where("id", id)
-    .update({
-      name: equipment.name,
-      inUse: equipment.inUse,
-      roomID: equipment.roomID,
-    })
-    .then(async () => {
-      await updateTrainingModules(id, equipment.trainingModules);
-    });
+      .where("id", id)
+      .update({
+        name: equipment.name,
+        inUse: equipment.inUse,
+        roomID: equipment.roomID,
+      })
+      .then(async () => {
+        await updateTrainingModules(id, equipment.trainingModules);
+      });
```

Comment:
I think there's something wrong with ur prettier config - this file has just been reformatted to 4 spaces per indent when it should be using 2

Reply:
I believe it has always been two for me, idk what change if it were changed...